### PR TITLE
Remove docker volumes once product-tests execution completes

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@ArturGajowy @ebd2

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -99,7 +99,7 @@ function stop_docker_compose_containers() {
     stop_application_runner_containers ${ENVIRONMENT}
 
     # stop containers started with "up"
-    environment_compose down
+    environment_compose down -v
```
Comment (ArturGajowy) [inline presto-product-tests/bin/run_on_docker.sh]: Change the comment to
`# stop containers started with "up", removing their volumes`

**Comment by ArturGajowy** [comment]
LGTM % minor comment

File: presto-product-tests/bin/run_on_docker.sh (hunk 1 above)
Comment (sanjay990) [reply presto-product-tests/bin/run_on_docker.sh]: Done

**Comment by ArturGajowy** [comment]
Will you be doing an upstream PR for that?
LOL it's merged already :D GJ!

**Pull request merged**